scripts/svm/closeDataWorkerLookUpTables.ts

```diff
@@ -0,0 +1,71 @@
+import * as anchor from "@coral-xyz/anchor";
+import { PublicKey, SendTransactionError } from "@solana/web3.js";
+import { AddressLookupTableProgram } from "@solana/web3.js";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+
+// Set up the provider
+const provider = anchor.AnchorProvider.env();
+anchor.setProvider(provider);
+const connection = provider.connection;
+const payer = provider.wallet.publicKey;
+
+// Parse arguments
+// const argv = yargs(hideBin(process.argv))
+//   .option("payer", { type: "string", default: payer.toBase58(), describe: "Payer public key" }).argv;
```

Comment:
are these redundant?

Reply:
yes, as only the payer (which is the unlocked account) can run this script as we need to send transactions from their address